torch/nn/intrinsic/qat/modules/conv_fused.py

```diff
@@ -303,6 +303,32 @@ def forward(self, input):
     def from_float(cls, mod):
         return super(ConvBnReLU1d, cls).from_float(mod)
 
+    def to_float(self):
```

Comment:
would it be possible to reuse code between 1d/2d/3d same as `from_float` is doing? Or is there some blocker

Reply:
probably not, let me try if I can combine them